runelite-client/src/main/java/net/runelite/client/plugins/keyremapping/KeyRemappingConfig.java

```diff
@@ -286,4 +287,15 @@ default ModifierlessKeybind space()
 	{
 		return new ModifierlessKeybind(KeyEvent.VK_SPACE, 0);
 	}
+
+	@ConfigItem(
+		position = 22,
+		keyName = "control",
+		name = "Control",
+		description = "The key which will replace {Control}."
+	)
+	default ModifierlessKeybind control()
+	{
+		return new ModifierlessKeybind(KeyEvent.VK_CONTROL, InputEvent.CTRL_DOWN_MASK);
```

Comment:
From testing on my system and talking to abex, it seems like this should be `new ModifierlessKeybind(KeyEvent.VK_UNDEFINED, InputEvent.CTRL_DOWN_MASK);`. Optionally we could just make this not have a default value and check for null, although we don't do that for the other keybinds.